docs/maui/extensions/servicecollection-extensions.md

```diff
@@ -0,0 +1,316 @@
+---
+title: ServiceCollectionExtensions - .NET MAUI Community Toolkit
+author: rjygraham
+description: "The ServiceCollectionExtensions provide a series of extension methods that simplify registering Views and ViewModels within a .NET MAUI  IServiceCollection."
+ms.date: 04/22/2022
+---
+
+<!-- markdownlint-configure-file { "MD024": { "allow_different_nesting": true } } -->
```

Comment:
Sorry I can't see why this was needed. What does this solve?

Reply:
Since each method has the same markdown headers, they were being flagged as duplicates without turning on the nesting.

Reply:
Oh thanks for clearing that up. I think we suffer from the same issue on other pages. Perhaps I need to get in touch with someone on the docs team and see if we can get this item disabled.